### PR TITLE
Coach can add availabilities

### PR DESCRIPTION
**Pull request by zdehkordi**
Coaches can now add their availabilities from their profile page. This is the first step in the build of our booking system.

### CODE REVIEW

File: app/controllers/user_controller.rb [hunk 1]
```diff
@@ -6,6 +6,31 @@ def booking
...
+    end
+    
+    
+    avail.coach_id = current_user.id
+
+    avail.save!
+    redirect_to availabilities_path
+  end
+
+  def avail_params
```
Comment (chenjoe569) [inline app/controllers/user_controller.rb]: This is probably not needed?

Comment (zdehkordi) [reply app/controllers/user_controller.rb]: oh you right, removing now and re comiitting

**Comment by chenjoe569** [comment]
Looks good. Seems like everything runs and testing passes

**Comment by chenjoe569** [comment]
Looks good to me. All tests passed and this one just had that extra function deleted - no effect on anything.

**Pull request merged**